nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -29,7 +29,7 @@
     clip_grad_norm_fp32,
 )
 from nemo.collections.nlp.modules.common.megatron.megatron_init import initialize_model_parallel_for_nemo
-from nemo.collections.nlp.modules.common.tokenizer_utils import get_nmt_tokenizer
+from nemo.collections.nlp.modules.common.tokenizer_utils import get_nmt_tokenizer, get_tokenizer
```

Comment:
since you have support of music tokenizer in the `get_nmt_tokenizer` method. there is no need to change this file. could you revert it back?